### ISSUE TITLE
Error when sync KV cert to Kubernetes TLS secrets

### ISSUE THREAD

**Issue by jsukhabut** [Azure/secrets-store-csi-driver-provider-azure#136]
**What steps did you take and what happened:**
Using secret object sync to sync certificate in KV to kubernetes tls secret as per instructions

```
objects:  |
      array:
        - |
          objectName: SSLCert
          objectType: secret
```

```
  secretObjects:
    - secretName: testsecret
      type: kubernetes.io/tls
      data:
      - objectName: SSLCert
        key: tls.key
      - objectName: SSLCert
        key: tls.crt
```

**What did you expect to happen:**
For secret testsecret to be create with tls.key and tls.crt

This was the error:

failed to get objectType:secret, objectName:SSLCert, objectVersion:: pkcs12: expected exactly two safe bags in the PFX PDU\"\n for pod: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx

**Which access mode did you use to access the Azure Key Vault instance:**
User Assigned Managed Identity

**Environment:**
- Secrets Store CSI Driver version: (use the image tag): v0.0.11
- Azure Key Vault provider version: (use the image tag):  0.0.6
- Kubernetes version: (use `kubectl version` and `kubectl get nodes -o wide`):  v1.16.9
- Cluster type: (e.g. AKS, aks-engine, etc): AKS

**Comment by aramase** [comment]
Multiple certificates are not support with pkcs12 in the current release of the provider (`0.0.6`). The PR to add this feature - https://github.com/Azure/secrets-store-csi-driver-provider-azure/pull/123 has been merged and will be available as part of next release `0.0.7` which is planned for this week.

**Comment by aramase** [comment]
Closing this issue with release 0.0.7 - https://github.com/Azure/secrets-store-csi-driver-provider-azure/releases/tag/0.0.7
